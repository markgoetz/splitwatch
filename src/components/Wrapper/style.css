.wrapper {
  padding: 0 20px;
}

@media (min-width: 769px) {
  .wrapper {
    width: 600px;
    margin: 0 auto;
  }
}