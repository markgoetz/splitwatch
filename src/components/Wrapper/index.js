import { h } from 'preact';
import style from './style.css';

export default function Wrapper(props) {
  return <div class={style.wrapper}>{props.children}</div>;
}