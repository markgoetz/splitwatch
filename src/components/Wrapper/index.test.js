import { h } from 'preact';
import Wrapper from './index';
import { shallow } from 'preact-render-spy';

describe('Wrapper', () => {

  test('Wrapper renders correctly', () => {
    const context = shallow(<Wrapper>
      <div>Test element</div>
      <button>Test button</button>
      <div>Extra test element</div>
    </Wrapper>);
    expect(context).toMatchSnapshot();
  });
});