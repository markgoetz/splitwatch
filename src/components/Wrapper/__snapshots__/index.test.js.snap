// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper Wrapper renders correctly 1`] = `
preact-render-spy (1 nodes)
-------
<div>
  <div>Test element</div>
  <button>Test button</button>
  <div>Extra test element</div>
</div>

`;
